wagtail/images/templates/wagtailimages/images/view_toggle_button.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailadmin_tags i18n %}
+
+<div>
+    {% if view_mode == "list" %}
+        <a href="?view=grid">
+            <button
+                type="button"
+                class="w-filter-button"
+                aria-label="{% trans 'Toggle view' %}"
+                style="align-items: center; justify-content: center; display: flex"
```

Comment:
This will need to be done with vanilla CSS or Tailwind utility classes, we can’t use inline styles.